### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by christianbiring1**
In this project, I had to connect my React-Redux application to an existing API to create and remove books on a remote server.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,52 +1,73 @@
+import axios from 'axios';
+
```
Comment (Ismailco) [inline src/redux/books/books.js:2]: - [ ] `redux-thunk` should be added to the project as a package and used as middleware, kindly implement this
e.g.

     ```
      import { createStore, applyMiddleware } from 'redux';
      import thunk from 'redux-thunk';
      import rootReducer from './reducers/index';
      
          const store = createStore(
              rootReducer,
              applyMiddleware(thunk)
          );
     ```

File: src/redux/books/books.js [hunk 2]
```diff
@@ -1,52 +1,73 @@
...
 const ADD_BOOK = 'ADD_BOOK';
 const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (Ismailco) [inline src/redux/books/books.js]: - [ ] [OPTIONAL] You can use the `ducks pattern` as a value for the actions, this is a best practice and a requirement in the past milestone 
e.g.
              ```
              const ADD_BOOK = bookstore/books/ADD_BOOK
              const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
              ```

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi  @christianbiring1,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (christianbiring1) [reply src/redux/books/books.js:2]: Hey @Ismailco!
I was trying to use the `createStore()` but in my IDEditor, I was receiving a warning like this method is deprecated we suggest you use `ConfigureStore()`. and there's a line through it. I mean the createStore. (~~createStore~~)

File: src/redux/books/books.js (hunk 2 above)
Comment (christianbiring1) [reply src/redux/books/books.js]: :rocket:  Alright
@Ismailco

File: src/redux/books/books.js (hunk 1 above)
Comment (Ismailco) [reply src/redux/books/books.js:2]: @christianbiring1, I think you can use it with ConfigureStore() like this:

```
const store = configureStore({
  rootReducer,
  applyMiddleware: thunk
})
```

**Review by descholar-ceo — APPROVED** [review]
Hi @christianbiring1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:+1: You have implemented the requested changes :heavy_check_mark: 
:+1: Great communication with reviewers :heavy_check_mark: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**